source/docs/create_project.md

```diff
@@ -28,3 +28,30 @@ $ cd &lt;YourDAppName&gt;
 </code></pre>
 
 This will create a simple project with all components disabled except the blockchain/contracts component.
+
+
+### Creating a project from a template
+
+If you want to start from a pre-existing template, then you can use the `--template` option:
```

Comment:
Are there any other pre-existing templates besides `angular`?

Reply:
There is another story about adding them, once added maybe we can add a link to it?